pombola/kenya/views.py

```diff
@@ -45,6 +50,19 @@
             'g': ('m', 'f'),
             'agroup': ('under', 'over'),
         },
+    },
+    'youth-employment-bill': {
+        'session_key_prefix': 'MIT3',
+        'base_view_name': 'youth-employment',
+        'pageview_label': 'youth-employment',
+        'template_prefix': 'youth',
+        'experiment_key': settings.YOUTH_EMPLOYMENT_BILL_EXPERIMENT_KEY,
+        'qualtrics_sid': 'SV_5hhE4mOfYG1eaOh',
```

Comment:
It's probably best to set this to `None` until we have the real new survey ID, just so we don't forget to update it. (Or was it clear that they'd be using the same survey for this experiment too?)